api-schema/xcherry.yaml

```diff
@@ -507,21 +463,25 @@ components:
           description: for async state API only, state id + sequence number
           type: string
         recoverFromApi: # if last execution failed, WAIT_UNTIL or EXECUTE, otherwise NONE
-          $ref: '#/components/schemas/StateApiType'
-    StateApiType:
+          $ref: '#/components/schemas/WorkerApiType'
+    WorkerApiType:
       type: string
       enum:
         - WAIT_UNTIL_API
         - EXECUTE_API
+        - RPC_API
     WorkerErrorResponse:
       type: object
       required: 
         - errorType
       properties:
         detail:
           type: string
+          description: "an optional field to let application set some detailed information. 
+          Default to the error message + stacktrace of the error"
         errorType:
           type: string
+          description: "an optional field for error handling. Default to the class/error Name"
```

Comment:
@zklgame following our discussion for RPC/State error handling, WDYT that we just keep it simple for now to use these two fields. 
For Java, the errorType is just the exception class name, the details is the message+stacktrace. 
For java application, I belive use the class name for error handling is mostly enough (and they can create new class for that purpoe)

Reply:
Sounds good!